StatusAPIs.py

```diff
@@ -0,0 +1,87 @@
+import requests
+import json
+
+
+class StatusAPIs:
+
+    def __init__(self, sentry_id, camera, API_KEY, url):
+        self.headers = {
+            'content-type': "application/json",
+            'x-api-key': API_KEY,
+            'cache-control': "no-cache"
+        }
+        self.url = url
+        self.sentry_id = sentry_id
+        self.camera = camera
+        self.payload = {'sentry_id': sentry_id, "camera_information": ""}
+
+    def get_results_for_status_api(self):
+        self.url += "?sentry_id=" + self.sentry_id + "&camera=" + camera
+        response = requests.request("GET", self.url, headers=self.headers, timeout=25)
+        return response
+
+    def update_payload_with_more_info(self, camera_id, camera_response):
+        cam_info = {camera_id: camera_response}
+        self.payload.update({"camera_information": cam_info})
+
+    def post_response_for_status_api(self):
+        response = requests.request("POST", self.url, data=json.dumps(self.payload), headers=self.headers, timeout=25)
+        return response
+
+    def parse_output_post(self, response):
+        output = response.text
+        print("The result after DB update is = ", output)
```

Comment:
Don't we need to parse output here ? What is the output

Reply:
The lambda is currently just returning "updated" keyword as response